www/src/pages/foundations/colors.jsx

```diff
@@ -144,7 +144,7 @@ export default function ColorsPage({ data }) {
           $brand-200
         </code>
 
-        <h6>Mixin (deprecated)</h6>
+        <h3>Mixin (deprecated)</h3>
```

Comment:
This should likely be the same heading level as "Variable name" above, as an `h4`, since it's still referring to SCSS usage.